theme/icons.go

```diff
@@ -9,6 +9,142 @@ import (
 	"fyne.io/fyne"
 )
 
+var (
+	// Icons specifies each of the known icon names that a theme can contain.
+	//
+	// Since 2.0.0
+	Icons = struct {
```

Comment:
What is the purpose of this struct?
If it’s just for providing constants, we should actually _use_ constants instead.

AFAICS, the struct approach allows to
```go
theme.Icons.Cancel = "confirm"
```
and that might not be intended.

Reply:
You're right we certainly don't want those remapped.
It was an attempt to avoid the explosion of const names without much namespace, but perhaps it was not the right approach.
As it was following the colour and size naming I will need to go back and revise them as well.